tensorflow/tools/dockerfiles/dockerfiles/devel-cpu.Dockerfile

```diff
@@ -108,3 +108,4 @@ RUN mkdir /bazel && \
 
 COPY bashrc /etc/bash.bashrc
 RUN chmod a+rwx /etc/bash.bashrc
+RUN chmod a+w /etc/passwd /etc/group
```

Comment:
I don't like this change, from a security point of view. Permissions on these two files should not be changed.

Instead, I'd recommend editing those files inside the docker container, manually or just switching to Python 3.

Reply:
On the other hand, this is only on a docker container running locally. But it's still making me uneasy.

Reply:
I'm okay with this since it's a locally-running docker container. Most of the time, docker containers run as root anyway, but ours don't (well, they're not supposed to), so the potential damage seems minimal.

Reply:
I agree that the damage is minimal. Not opposed to the change, was just voicing concern.